chroma-manager/tests/integration/utils/chroma_log_collector.py

```diff
@@ -10,8 +10,67 @@
 import json
 import re
 import sys
-
-from chroma_common.lib.shell import Shell
```

Comment:
it gets used before we install chroma packages and before we install pip/virtual environments, this used to work because we had a symlink to the source of chroma-common, now that is no longer valid